antora-playbook-staging.yml

```diff
@@ -41,207 +94,266 @@ site:
+      branches:
+        - main
+    - url: https://github.com/couchbase/docs-kafka
+      branches:
+        - release/4.3
+        - release/4.2
+    - url: https://github.com/couchbase/docs-spark
+      branches:
+        - release/3.5
+        - release/3.3
+    - url: https://github.com/couchbase/docs-tableau
+      branches:
+        - release/1.1
+        - release/1.0
+    - url: https://github.com/couchbase/docs-connectors-power-bi
+      branches:
+        - main
+    - url: https://github.com/couchbase/docs-connectors-superset.git
+      branches:
+        - release/1.0
+    - url: https://github.com/couchbase/docs-connectors-talend
+      branches:
+        - master
+    - url: https://github.com/couchbaselabs/docs-enterprise-analytics
+      branches:
+        - release/2.0
+    - url: https://github.com/couchbase/docs-analytics
+      branches:
+        - release/7.6
+        - release/7.2
+        - release/7.1
+    - url: https://github.com/couchbase/couchbase-cli
+      branches:
+        - trinity
+        - neo
+        - 7.1.x-docs
+      start_path: docs
+    - url: https://github.com/couchbase/backup
+      branches:
+        - trinity
+        - neo
+        - 7.1.x-docs
+      start_path: docs
+    - url: https://github.com/couchbase/docs-server
+      branches:
+        - release/7.6
+        - release/7.2
+        - release/7.1
+    - url: https://github.com/couchbase/docs-sdk-common
+      branches:
+        - release/8.0
+        - release/7.7
+        - release/7.6.6
+    - url: https://github.com/couchbase/docs-sdk-c
+      branches:
+        - release/3.3
+    - url: https://github.com/couchbase/docs-sdk-cxx
+      branches:
+        - release/1.1
+        - release/1.0
+    - url: https://github.com/couchbase/docs-sdk-dotnet
+      branches:
+        - temp/3.8
+        - temp/3.7
+        - temp/3.6
+    - url: https://github.com/couchbase/docs-efcore
+      branches:
+        - release/1.0
+    - url: https://github.com/couchbase/docs-sdk-go
+      branches:
+        - temp/2.11
+        - temp/2.10
+        - temp/2.9
+    - url: https://github.com/couchbase/docs-sdk-java
+      branches:
+        - release/3.9
+        - release/3.8
+        - temp/3.7
+    - url: https://github.com/couchbase/docs-quarkus-extension
+      branches:
+        - release/1.1
+        - release/1.0
+    - url: https://github.com/couchbase/docs-sdk-kotlin
+      branches:
+        - release/3.9
+        - temp/1.5
+        - temp/1.4
+        - temp/1.3
+        - release/1.2
+    - url: https://github.com/couchbase/docs-sdk-scala
+      branches:
+        - release/3.9
+        - release/1.8
+        - release/1.7
+    - url: https://github.com/couchbase/docs-sdk-nodejs
+      branches:
+        - temp/4.5
+        - temp/4.4
+        - temp/4.3
+    - url: https://github.com/couchbase/docs-sdk-php
+      branches:
+        - temp/4.3
+        - temp/4.2
+        - release/4.1
```

Comment:
PHP 4.1 has just gone EOL - I'll be removing from the main playbook later.

Reply:
great, that should update automagically the next time I run the script...